src/Compilers/CSharp/Portable/Parser/LanguageParser_InterpolatedString.cs

```diff
@@ -187,10 +188,24 @@ ReadOnlySpan<char> getIndentationWhitespace(ReadOnlySpan<char> originalTextSpan)
                             var isLegalBlankLine = isBlankLine && indentationWhitespace.StartsWith(currentLineWhitespace);
                             if (!isLegalBlankLine)
                             {
-                                indentationError ??= MakeError(
-                                    lineStartPosition,
-                                    width: currentIndex - lineStartPosition,
-                                    ErrorCode.ERR_LineDoesNotStartWithSameWhitespace);
+                                // Specialized error message if this is a spacing difference.
+                                if (CheckForSpaceDifference(
```

Comment:
nit: given that this method is pretty large already, consider extracting the entire `if` statement (then we can have `MakeRawStringIndentationError` or some such, instead of `CheckForSpaceDifference`)

Reply:
Never mind. I see that `CheckForSpaceDifference` is also used elsewhere